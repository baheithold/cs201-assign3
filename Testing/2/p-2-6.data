27 47 64 ; 46 1 98 ; 5 34 91 ; 2 50 179 ; 2 27 206 ; 
20 22 442 ; 9 54 182 ; 23 48 387 ; 25 55 107 ; 3 45 219 ; 
51 1 55 ; 52 3 282 ; 22 22 8 ; 9 34 65 ; 36 16 51 ; 
59 54 481 ; 49 29 205 ; 34 52 170 ; 25 51 201 ; 6 59 403 ; 
7 54 422 ; 11 12 341 ; 33 16 393 ; 3 26 76 ; 20 37 401 ; 
10 51 481 ; 39 20 364 ; 28 63 482 ; 15 0 4 ; 10 27 417 ; 
0 58 0 ; 12 11 19 ; 29 17 222 ; 55 45 52 ; 28 18 187 ; 
15 27 187 ; 36 32 494 ; 35 22 141 ; 35 46 369 ; 63 31 285 ; 
57 23 107 ; 4 23 96 ; 21 33 180 ; 2 13 425 ; 20 61 44 ; 
48 0 441 ; 16 14 96 ; 38 4 68 ; 20 21 27 ; 51 54 27 ; 
11 34 470 ; 34 14 483 ; 3 6 55 ; 17 28 285 ; 14 40 395 ; 
14 17 140 ; 29 10 204 ; 33 38 100 ; 54 38 366 ; 45 9 302 ; 
15 43 325 ; 29 6 492 ; 35 37 470 ; 63 26 482 ; 40 18 283 ; 
57 30 61 ; 3 26 3 ; 42 38 308 ; 16 24 117 ; 25 30 152 ; 
4 23 197 ; 10 28 212 ; 47 2 293 ; 10 44 324 ; 28 39 28 ; 
58 48 4 ; 20 8 489 ; 58 60 285 ; 18 18 311 ; 48 30 391 ; 
7 23 404 ; 35 31 24 ; 38 4 382 ; 18 12 385 ; 58 20 319 ; 
42 4 431 ; 50 46 9 ; 47 27 461 ; 1 50 35 ; 31 25 301 ; 
54 25 357 ; 21 29 85 ; 25 43 476 ; 37 40 2 ; 58 48 497 ; 
62 19 95 ; 44 49 5 ; 7 34 288 ; 57 14 399 ; 18 11 308 ; 
43 57 19 ; 8 46 229 ; 51 6 150 ; 28 5 279 ; 12 22 440 ; 
31 33 14 ; 16 22 5 ; 51 62 253 ; 1 21 397 ; 12 45 246 ; 
5 36 193 ; 51 17 492 ; 58 19 373 ; 63 46 12 ; 21 47 148 ; 
54 41 452 ; 12 34 208 ; 11 8 220 ; 32 53 7 ; 14 27 252 ; 
50 20 257 ; 4 57 82 ; 23 38 314 ; 6 6 280 ; 53 3 335 ; 
30 11 328 ; 1 51 2 ; 9 44 66 ; 62 23 309 ; 25 4 130 ; 
46 61 40 ; 39 59 410 ; 13 45 266 ; 19 25 314 ; 22 4 128 ; 
34 4 205 ; 21 58 207 ; 1 61 344 ; 25 10 17 ; 29 56 475 ; 
26 52 317 ; 21 35 383 ; 3 21 222 ; 28 31 3 ; 33 47 442 ; 
37 52 41 ; 32 24 251 ; 29 60 280 ; 39 42 202 ; 32 21 227 ; 
20 26 404 ; 55 13 194 ; 13 39 327 ; 44 14 322 ; 28 9 459 ; 
16 26 424 ; 40 13 45 ; 37 21 177 ; 15 31 386 ; 36 2 334 ; 
63 2 150 ; 12 28 374 ; 52 12 122 ; 2 62 159 ; 11 13 151 ; 
37 61 375 ; 50 19 240 ; 8 8 304 ; 39 27 61 ; 42 21 232 ; 
44 47 441 ; 9 29 208 ; 21 3 212 ; 19 54 380 ; 33 22 67 ; 
30 17 214 ; 49 29 364 ; 58 18 470 ; 21 43 110 ; 42 20 42 ; 
25 33 288 ; 54 45 456 ; 58 5 389 ; 48 18 118 ; 6 33 466 ; 
24 59 243 ; 53 59 373 ; 7 46 255 ; 51 40 65 ; 7 10 42 ; 
40 22 20 ; 21 15 149 ; 26 16 7 ; 44 39 365 ; 13 17 438 ; 
9 28 172 ; 4 5 296 ; 50 32 254 ; 27 53 497 ; 37 31 88 ; 
60 51 315 ; 11 32 119 ; 27 4 356 ; 2 33 91 ; 19 47 407 ; 
47 4 169 ; 53 48 294 ; 21 18 498 ; 11 60 401 ; 42 24 316 ; 
30 60 433 ; 62 7 245 ; 2 40 208 ; 36 35 212 ; 19 10 374 ; 
23 63 366 ; 8 61 77 ; 26 0 234 ; 22 53 296 ; 47 5 475 ; 
43 58 369 ; 50 44 443 ; 26 40 402 ; 61 0 252 ; 8 27 8 ; 
7 9 312 ; 5 38 483 ; 5 57 23 ; 58 33 334 ; 63 20 116 ; 
57 49 107 ; 37 45 326 ; 14 39 393 ; 14 35 311 ; 41 4 229 ; 
50 56 349 ; 25 3 192 ; 18 23 109 ; 51 5 394 ; 7 9 300 ; 
56 61 495 ; 37 27 265 ; 12 44 2 ; 48 24 120 ; 52 1 331 ; 
44 50 8 ; 47 42 371 ; 6 40 144 ; 11 42 414 ; 21 39 477 ; 
18 34 160 ; 45 23 332 ; 26 50 142 ; 50 26 490 ; 51 17 130 ; 
37 50 274 ; 15 5 234 ; 55 49 198 ; 34 47 484 ; 9 32 336 ; 
43 24 165 ; 3 8 283 ; 53 30 16 ; 15 1 462 ; 33 19 80 ; 
19 26 451 ; 24 12 276 ; 9 46 248 ; 56 23 475 ; 24 11 353 ; 
48 20 34 ; 56 47 366 ; 23 55 261 ; 24 26 426 ; 43 46 333 ; 
5 54 314 ; 18 34 81 ; 0 30 153 ; 23 18 26 ; 34 39 262 ; 
54 6 394 ; 37 16 26 ; 29 29 399 ; 55 31 450 ; 37 48 374 ; 
27 35 153 ; 62 40 63 ; 28 6 189 ; 46 44 81 ; 22 11 373 ; 
28 1 455 ; 44 40 438 ; 9 7 159 ; 40 61 136 ; 24 11 331 ; 
59 4 325 ; 36 3 272 ; 42 20 98 ; 22 38 255 ; 33 15 161 ; 
35 26 79 ; 11 28 339 ; 18 4 159 ; 15 8 152 ; 26 4 448 ; 
30 37 381 ; 34 33 412 ; 54 28 180 ; 28 51 134 ; 43 56 301 ; 
6 20 146 ; 34 11 65 ; 38 47 84 ; 47 27 102 ; 51 48 273 ; 
24 29 119 ; 57 5 146 ; 21 18 120 ; 8 8 26 ; 1 21 3 ; 
21 4 70 ; 32 21 388 ; 15 50 191 ; 43 32 45 ; 27 30 384 ; 
56 53 391 ; 61 43 391 ; 15 3 9 ; 24 18 46 ; 45 61 55 ; 
49 19 400 ; 7 48 161 ; 57 47 177 ; 25 20 461 ; 55 41 164 ; 
45 48 439 ; 24 16 315 ; 28 41 56 ; 46 23 196 ; 43 14 16 ; 
62 6 169 ; 46 19 7 ; 29 1 242 ; 50 62 311 ; 27 38 316 ; 
11 41 384 ; 27 36 485 ; 4 28 413 ; 28 36 289 ; 42 4 195 ; 
49 18 282 ; 4 17 121 ; 5 23 303 ; 3 22 21 ; 41 46 319 ; 
18 62 434 ; 54 59 405 ; 19 17 329 ; 55 2 361 ; 60 34 401 ; 
14 44 208 ; 31 53 329 ; 54 45 425 ; 12 2 379 ; 58 9 2 ; 
57 43 108 ; 52 24 338 ; 5 33 9 ; 8 47 28 ; 42 24 91 ; 
22 44 215 ; 12 49 69 ; 57 59 364 ; 59 10 490 ; 4 28 258 ; 
47 24 337 ; 7 39 462 ; 41 44 470 ; 24 4 72 ; 48 19 112 ; 
29 31 286 ; 14 56 376 ; 9 49 4 ; 19 31 177 ; 48 26 416 ; 
8 15 40 ; 47 61 187 ; 27 31 411 ; 31 7 244 ; 51 55 48 ; 
18 17 224 ; 10 2 333 ; 59 62 204 ; 27 52 261 ; 53 36 237 ; 
4 25 133 ; 1 60 261 ; 32 39 257 ; 40 59 332 ; 31 39 16 ; 
49 4 103 ; 51 61 105 ; 49 5 499 ; 37 23 95 ; 10 60 438 ; 
35 17 435 ; 31 46 423 ; 7 59 216 ; 2 36 284 ; 42 44 52 ; 
46 39 132 ; 43 52 13 ; 49 11 72 ; 8 30 313 ; 4 12 267 ; 
21 47 201 ; 3 22 304 ; 63 62 115 ; 35 7 46 ; 15 55 45 ; 
54 7 307 ; 42 8 261 ; 53 60 240 ; 20 34 97 ; 32 45 297 ; 
16 2 225 ; 38 58 366 ; 37 17 334 ; 44 48 385 ; 35 17 352 ; 
43 28 455 ; 51 21 93 ; 48 57 75 ; 18 15 400 ; 63 44 16 ; 
1 14 213 ; 59 4 425 ; 12 62 75 ; 60 55 232 ; 13 19 180 ; 
41 62 324 ; 62 31 295 ; 56 42 458 ; 7 43 386 ; 51 63 244 ; 
1 52 151 ; 6 57 104 ; 4 37 463 ; 60 53 492 ; 15 50 88 ; 
14 18 298 ; 45 29 279 ; 24 27 294 ; 3 10 286 ; 2 10 149 ; 
55 10 384 ; 48 10 218 ; 21 6 16 ; 11 14 316 ; 61 18 4 ; 
15 36 284 ; 44 32 110 ; 7 26 427 ; 17 12 311 ; 28 5 14 ; 
38 49 201 ; 48 31 288 ; 54 19 200 ; 4 12 366 ; 23 0 17 ; 
59 4 186 ; 27 22 345 ; 53 22 345 ; 1 45 440 ; 6 8 222 ; 
56 37 434 ; 23 57 80 ; 43 61 264 ; 55 23 303 ; 55 60 39 ; 
60 30 62 ; 18 15 22 ; 40 44 197 ; 22 17 87 ; 30 7 124 ; 
3 25 459 ; 60 17 179 ; 57 57 2 ; 17 24 93 ; 13 60 120 ; 
44 54 88 ; 59 44 280 ; 40 62 158 ; 57 5 3 ; 0 13 477 ; 
25 28 95 ; 42 35 58 ; 35 2 6 ; 60 11 491 ; 56 12 380 ; 
46 4 190 ; 27 56 310 ; 25 62 233 ; 30 45 157 ; 43 50 374 ; 
8 21 402 ; 43 59 231 ; 45 29 89 ; 57 57 241 ; 5 37 489 ; 
9 31 77 ; 1 5 363 ; 63 39 74 ; 45 48 407 ; 31 51 449 ; 
53 53 177 ; 48 17 345 ; 14 22 40 ; 7 63 62 ; 44 36 3 ; 
12 1 336 ; 17 13 1 ; 56 51 74 ; 41 51 417 ; 28 16 182 ; 
17 25 50 ; 34 42 263 ; 57 39 379 ; 56 17 405 ; 29 52 184 ; 
30 28 245 ; 43 13 481 ; 31 63 202 ; 18 48 143 ; 34 15 312 ; 
60 13 436 ; 38 0 269 ; 13 35 192 ; 30 21 145 ; 18 13 369 ; 
46 50 354 ; 59 35 29 ; 58 38 460 ; 43 33 371 ; 58 5 373 ; 
7 5 268 ; 7 22 232 ; 42 10 462 ; 63 55 0 ; 13 29 406 ; 
63 31 37 ; 35 0 79 ; 9 13 35 ; 42 56 477 ; 47 54 176 ; 
52 22 166 ; 10 50 145 ; 21 52 376 ; 12 4 443 ; 41 51 398 ; 
9 52 279 ; 9 35 331 ; 22 14 386 ; 15 52 147 ; 5 4 109 ; 
27 15 232 ; 13 32 8 ; 1 57 354 ; 6 60 315 ; 57 62 36 ; 
45 14 385 ; 16 29 70 ; 31 15 120 ; 19 10 198 ; 23 47 177 ; 
38 3 430 ; 7 11 429 ; 0 29 406 ; 60 5 176 ; 58 57 238 ; 
8 22 147 ; 37 61 496 ; 52 45 12 ; 62 51 231 ; 45 9 186 ; 
48 35 202 ; 59 25 296 ; 25 27 115 ; 30 31 387 ; 23 41 407 ; 
45 44 222 ; 42 28 462 ; 23 29 460 ; 11 60 452 ; 20 47 235 ; 
55 57 416 ; 16 41 285 ; 44 33 144 ; 11 24 405 ; 53 12 31 ; 
33 58 58 ; 62 48 224 ; 27 12 246 ; 23 29 450 ; 6 44 181 ; 
63 40 321 ; 41 21 300 ; 10 62 49 ; 34 63 325 ; 46 31 304 ; 
40 21 429 ; 25 57 495 ; 37 8 497 ; 2 46 152 ; 46 31 339 ; 
23 36 246 ; 44 32 277 ; 42 45 378 ; 42 38 16 ; 9 43 79 ; 
30 36 177 ; 24 63 34 ; 32 24 111 ; 14 12 220 ; 45 24 167 ; 
17 42 185 ; 49 59 340 ; 31 54 473 ; 5 58 23 ; 48 33 453 ; 
21 55 90 ; 20 53 197 ; 45 12 497 ; 57 8 237 ; 17 7 219 ; 
59 53 124 ; 55 57 477 ; 45 42 226 ; 39 6 421 ; 9 11 239 ; 
0 26 483 ; 53 7 436 ; 1 12 93 ; 9 49 302 ; 17 36 67 ; 
6 4 377 ; 63 53 32 ; 42 23 459 ; 48 25 36 ; 59 32 484 ; 
21 23 177 ; 28 36 497 ; 40 5 478 ; 25 19 83 ; 61 26 180 ; 
1 51 141 ; 54 8 299 ; 14 19 199 ; 39 12 461 ; 7 44 461 ; 
30 21 356 ; 2 62 34 ; 7 28 394 ; 26 59 435 ; 53 35 244 ; 
40 29 190 ; 48 28 99 ; 4 27 450 ; 16 4 320 ; 60 29 483 ; 
17 50 178 ; 15 28 240 ; 43 34 459 ; 38 17 278 ; 9 26 1 ; 
38 32 46 ; 3 39 290 ; 30 37 323 ; 35 49 399 ; 0 36 407 ; 
50 30 35 ; 14 55 108 ; 49 18 327 ; 2 52 318 ; 28 49 16 ; 
60 35 101 ; 36 62 68 ; 9 9 215 ; 59 24 389 ; 31 55 404 ; 
62 18 425 ; 53 62 455 ; 7 57 30 ; 59 31 247 ; 45 43 189 ; 
16 0 454 ; 14 60 230 ; 23 59 118 ; 47 25 420 ; 38 25 455 ; 
56 26 58 ; 54 14 56 ; 47 16 245 ; 14 3 232 ; 57 56 148 ; 
58 10 493 ; 54 25 461 ; 50 11 400 ; 11 7 279 ; 36 62 147 ; 
63 21 465 ; 13 41 270 ; 29 51 168 ; 32 3 488 ; 24 15 250 ; 
35 8 268 ; 60 18 198 ; 7 42 404 ; 15 25 306 ; 13 56 441 ; 
34 62 418 ; 12 60 309 ; 63 48 274 ; 2 20 491 ; 18 7 59 ; 
26 51 19 ; 44 25 326 ; 22 53 358 ; 48 4 290 ; 40 29 337 ; 
38 27 306 ; 34 24 109 ; 19 34 301 ; 39 9 128 ; 46 16 378 ; 
33 48 348 ; 58 50 383 ; 48 49 443 ; 52 55 134 ; 17 12 382 ; 
45 46 38 ; 5 51 69 ; 40 4 391 ; 49 20 378 ; 2 39 18 ; 
50 11 60 ; 36 10 478 ; 21 54 116 ; 12 56 232 ; 24 42 484 ; 
7 4 166 ; 58 5 491 ; 63 41 176 ; 19 31 292 ; 58 61 182 ; 
6 10 108 ; 16 56 90 ; 6 32 454 ; 62 57 60 ; 40 5 400 ; 
45 35 109 ; 50 46 196 ; 27 63 270 ; 59 55 133 ; 56 33 117 ; 
2 13 247 ; 58 43 77 ; 27 46 138 ; 20 6 372 ; 25 42 471 ; 
61 39 463 ; 43 43 92 ; 42 42 223 ; 34 39 302 ; 3 56 357 ; 
17 4 258 ; 60 21 416 ; 42 55 311 ; 48 32 410 ; 27 7 446 ; 
2 22 107 ; 46 19 208 ; 24 2 444 ; 63 12 490 ; 56 18 140 ; 
60 16 76 ; 17 16 109 ; 9 11 477 ; 41 33 340 ; 48 11 256 ; 
6 15 127 ; 25 15 414 ; 27 59 312 ; 40 18 444 ; 58 10 295 ; 
10 6 335 ; 26 19 311 ; 38 29 153 ; 7 17 146 ; 19 53 367 ; 
34 53 435 ; 49 55 369 ; 45 39 411 ; 63 47 282 ; 9 54 138 ; 
16 54 188 ; 35 49 42 ; 0 62 265 ; 17 20 223 ; 7 19 122 ; 
60 2 261 ; 51 56 99 ; 26 7 345 ; 9 49 294 ; 63 4 437 ; 
53 16 414 ; 39 39 154 ; 37 24 118 ; 58 47 430 ; 
