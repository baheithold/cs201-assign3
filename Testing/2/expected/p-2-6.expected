0: 27
1: 8(27)8 39(27)61
2: 2(39)18 31(39)16 47(8)28
3: 28(31)3 33(31)14 35(2)6
4: 5(33)9 41(28)56 49(28)16 59(35)29
5: 9(49)4 11(49)72 44(49)5 57(5)23
6: 7(57)30 12(44)2 26(9)1 34(9)65 36(44)3 43(57)19 58(9)2 62(57)36 63(44)16
7: 0(58)0 14(43)16 24(63)34 46(63)12 48(58)4 51(26)19 52(43)13 53(12)31 55(63)0 56(26)58
8: 15(0)4 19(46)7 20(48)34 23(0)17 25(48)36 30(53)16 32(53)7 45(52)12 50(46)9 54(51)27 61(46)40
9: 1(50)35 3(15)9 10(25)17 13(32)8 17(25)50 18(61)4 21(20)27 38(32)46 42(20)42
10: 4(38)68 6(21)16 22(3)21
11: 16(22)5 40(22)20
12: 29(16)70 37(40)2 60(16)76
weight: 1454
